cdap-storage-ext-spanner/src/main/java/io/cdap/cdap/storage/spanner/SpannerStructuredTable.java

```diff
@@ -388,6 +389,16 @@ public void deleteAll(Range range) throws InvalidFieldException {
     transactionContext.executeUpdate(builder.build());
   }
 
+  @Override
+  public void scanDeleteAll(Range keyRange)
```

Comment:
Is not Spanner SQL? It should be trivial to support it, similar to PostgreSQL.

Reply:
The initial idea was we'd only add the support for CDF and hence only cloud SQL. But yes its simple enough and should work on spanner so added the logic here as well.

Reply:
Well, there are plans to migrate CDF to Spanner. It would be easier to add support now than catch it later

Reply:
Ok, I had anyway added the code for spanner as well today. PTAL. Resolving this comment for now. Feel free to reopen if needed.